tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/XUnitEvpTests.cs

```diff
@@ -74,13 +74,12 @@ public static IEnumerable<object[]> GetDataForEarlyFlakeDetection()
         }
     }
 
-    [SkippableTheory]
-    [MemberData(nameof(GetData))]
-    [Trait("Category", "EndToEnd")]
-    [Trait("Category", "TestIntegrations")]
-    public async Task SubmitTraces(string packageVersion, string evpVersionToRemove, bool expectedGzip)
+    public virtual async Task SubmitTraces(string packageVersion, string evpVersionToRemove, bool expectedGzip)
     {
+        SetEnvironmentVariable(ConfigurationKeys.DebugEnabled, "1");
+
```

Comment:
Probably shouldn't enable this
```suggestion
```

Reply:
done in [5dc9c9d](https://github.com/DataDog/dd-trace-dotnet/pull/5634/commits/5dc9c9d4f26816d8a22caa72ceeeb3807286fd08)